util/xxh3p.h

```diff
@@ -253,7 +253,9 @@ XXH_FORCE_INLINE U64x2 XXH_vec_mule(U32x4 a, U32x4 b) {
 #if defined(XXH_NO_PREFETCH)
 #  define XXH_PREFETCH(ptr)  (void)(ptr)  /* disabled */
 #else
-#  if defined(_MSC_VER) && (defined(_M_X64) || defined(_M_I86))  /* _mm_prefetch() is not defined outside of x86/x64 */
+#if defined(_MSC_VER) && \
+    (defined(_M_X64) ||  \
+     defined(_M_IX86)) /* _mm_prefetch() is not defined outside of x86/x64 */
```

Comment:
@Cyan4973 This compilation bug (_M_I86) is still upstream also https://github.com/Cyan4973/xxHash/blob/dev/xxhash.h#L2484

Reply:
Thanks for the pointer @pdillinger !

My understanding of this bug is that,
under `x86` (32-bit) compilation on MSVC,
it would disable prefetching, hence lose performance,

but nothing would crash at runtime, nor fail compilation,
neither on `x86` nor on any other target ?

The fix is straightforward,
I'm just wondering is there is any relevant test to add in CI
that would have been able to detect such an issue.

Reply:
The reason I found it was because I grepped the entire tree for _M_X64 to find x64-isms as part of my goal to get the code to build on ARM64 MSVC targets.